ReactAndroid/src/main/java/com/facebook/react/views/text/ReactInlineBorderSpan.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.views.text;
+
+import android.graphics.Paint;
+import android.text.style.LineBackgroundSpan;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Color;
+
+/**
+ * We use a custom {@link LineBackgroundSpan}, because RN Text component can't render borders. Details here:
+ * https://github.com/Expensify/App/issues/4733
+ */
+public class ReactInlineBorderSpan implements LineBackgroundSpan, ReactSpan {
+  private final int effectiveStart;
+  private final int effectiveEnd;
+  private final int effectiveFontSize;
+
+  private final int backgroundColor = Color.parseColor("#f6f6f6");
+  private final int borderColor = Color.parseColor("#b3b8bc");
+  private final int borderRadius = 8;
+  private final int borderWidth = 2;
+
+  public ReactInlineBorderSpan(int effectiveFontSize, int effectiveStart, int effectiveEnd) {
+    this.effectiveFontSize = effectiveFontSize;
+    this.effectiveStart = effectiveStart;
+    this.effectiveEnd = effectiveEnd;
+  }
+
+  /**
+   * Calculates text start and end position per given lines text range
+   */
+  private int[] calculateBorderedTextCharRange(int lineStart, int lineEnd) {
+    /**
+     * If bordered text's position falls in lines size range return the position of bordered text's first char
```

Comment:
If bordered text's position falls within the lines size range...

Reply:
fixed